src/features/MovieCategories/component.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import './style.scss';
+
+export default class MovieCategories extends Component 
+{
+    // define our state
+    constructor(props)
+{
+        super(props);
+    }
+
+    render() 
+{
+        return (
+	<aside className='movieSection-categories'>
+		<ul className='category-items'>
+			{this.props.categories.map((item, index) => (
+				<h4 
+					key={index} 
+					className={item.active ? 'category-item active' : 'category-item'}
+					onClick={(e) => this.props.handleChangeCategory(e)}
```

Comment:
don't use anonymous functions right in jsx, create callbacks in component methods instead. Same as with map, current syntax will force react to create this callback function on every render, when it simply can be reused